app/views/content/how-to-apply/teacher-training-application.md

```diff
@@ -32,9 +31,9 @@ external_content:
     - content/train-to-be-a-teacher/promos/apply-for-your-course
 promo_content:
     - content/train-to-be-a-teacher/promos/adviser-promo-apply
-navigation: 20.25
-navigation_title: How to apply
-navigation_description: Discover tips on preparing your teacher training application, from writing your personal statement to choosing your references.
+navigation: 30.05
+navigation_title: Teacher training application
+navigation_description: Find out what goes in your teacher training application and when to apply.
```

Comment:
I presume we'll look to split out the `when to apply` on a different card? Do we have a ticket in the backlog for this - might be useful to get in place before start of cycle (next sprint?)?